crates/oxc_transformer/src/react_jsx/mod.rs

```diff
@@ -133,29 +141,33 @@ impl<'a> ReactJsx<'a> {
     fn add_import_jsx(&mut self) {
         if !self.import_jsx {
             self.import_jsx = true;
-            self.add_import_statement("jsx", "_jsx", "react/jsx-runtime");
+            self.add_import_statement("jsx", "_jsx", &self.jsx_runtime_importer.clone());
```

Comment:
All these clone allocate, which hinder performance, we should try and remove it.

Reply:
I still haven't found a way to remove `.clone` due to borrowing rule restrictions.

```shell
error[E0502]: cannot borrow `*self` as mutable because it is also borrowed as immutable
   --> crates/oxc_transformer/src/react_jsx/mod.rs:154:13
    |
154 |             self.add_import_statement("jsxs", "_jsxs", &self.jsx_runtime_importer);
    |             ^^^^^--------------------^^^^^^^^^^^^^^^^^^--------------------------^
    |             |    |                                     |
    |             |    |                                     immutable borrow occurs here
    |             |    immutable borrow later used by call
    |             mutable borrow occurs here


```

Reply:
I tried another way to fix this problem at https://github.com/web-infra-dev/oxc/pull/1115/commits/2706388f6d02fc0f5162af2b4db95d35b4cc9d08. But this way will increase boilerplate code

Reply:
I have a question about this `clone` here, with Cow, will Cow<st>.clone() result the Owned String being duplicated?

Reply:
As I understand it, yes.

Reply:
If the Cow.clone() will duplicate the owned String, I think these codes still have an impact on the performance?

Reply:
Yes, I found another fix. You can check the latest changes. if it still affects performance, you can let me know!